### PR TITLE
CA-211529: Corrected tapdisk stats definitions

### PR DESCRIPTION
**Pull request by chandrikas**
Signed-off-by: Chandrika Srinivasan <chandrika.srinivasan@citrix.com>

### CODE REVIEW

File: drivers/tapdisk-metrics-stats.h [hunk 1]
```diff
@@ -25,15 +25,15 @@
 
 struct stats {
     uint32_t version;
-    unsigned long long oo_reqs;
```
Comment (andyhhp) [inline drivers/tapdisk-metrics-stats.h]: You need a uint32_t _pad0; here, or this structure still has different alignment between 32bit and 64bit compiles

**Comment by chandrikas** [comment]
@andyhhp: I have added another commit. Will squash them if they look alright.

**Comment by andyhhp** [comment]
LGTM

**Comment by chandrikas** [comment]
@andyhhp: turns out I need another minor change in td-req.c. Please check.

**Comment by andyhhp** [comment]
Still LGTM, and apparently the compiler now agrees as well :)

**Pull request closed without merging**